app/src/main/java/io/gnosis/safe/ui/settings/owner/OwnerSeedPhraseFragment.kt

```diff
@@ -85,6 +92,15 @@ class OwnerSeedPhraseFragment : BaseViewBindingFragment<FragmentOwnerSeedPhraseB
         })
     }
 
+    private fun replaceDoubleNewlineWithParagraphLineSpacing(@StringRes stringResource: Int): SpannableString {
```

Comment:
why don't we just use two TextView's without doing some text manipulations at runtime?

Reply:
Because this will work with future layouts when the text has paragraphs. "\n\n" can be replaced by a paragraph line spacing. From Sasha's answer I suspect that this will happen more often.

Reply:
Ok, But the value in size span (8) should come from dimens

Reply:
done